src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,16 @@
+// your components will all go in this `component` directory.
+// feel free to change this component.js into TodoList.js
+import React, { useState } from 'react';
+
+export default class TodoForm extends React.Component {
+
```

Comment:
If no state or side effects is involved in a component, you can use function components instead.